src/parcel/ParcelView.java

```diff
@@ -126,78 +140,150 @@ static boolean inputParcelInformation() {
 
+        return checkProduct;
+    }
 
-        if (selectExit.toUpperCase().contains("T")) {
-            System.out.println("택배가 접수되었습니다.");
-            // 보내는 분
-            SenderAddress newsenderaddress = new SenderAddress(SenderAddressStateProvinceRegion, SenderAddressCity, SenderAddressStreetAddress, SenderAddressZipPostalCode);
-            Sender newsender = new Sender(SenderName, newsenderaddress, SenderNumber);
+    static Parcel saveParcel (String newTrackingNumber, Sender newsender, Recipient newrecipient, ProductInfo newproductinfo){
 
-            // 받는 분
-            RecipientAddress newrecipientAddress = new RecipientAddress(RecipientAddressStateProvinceRegion, RecipientAddressCity, RecipientAddressStreetAddress, RecipientAddressZipPostalCode);
-            Recipient newrecipient = new Recipient(RecipientName, newrecipientAddress, RecipientNumber);
+        // 입력된 데이터 기반으로 하나의 객체 생성
+        Parcel newParcel = new Parcel(newTrackingNumber, newsender, newrecipient, newproductinfo, "접수완료", 1000);
+        return newParcel;
+    }
 
-            // 상품 정보
-            ProductInfo newproductinfo = new ProductInfo(productName, productValue, productSize);
 
-            // 운송장번호
-            // 보내는 분 지역번호 3자리
-            String SenderLocalNumber = locals.getCode();
-            // 받는 분 지역번호 3자리
-            String RecipientLocalNumber = locals1.getCode();
-            // 남은 뒷자리 4자리는 배열에 남은 택배길이
-            String lastNumber = String.format("%04d", repository.getParcelArray().length + 1);
+    static void receiptParcel(boolean checkSender, boolean checkRecipient, boolean checkProduct, Parcel newParcel) {
 
-            // 최종 운송장 번호
-            String Number = SenderLocalNumber.concat(RecipientLocalNumber).concat(lastNumber);
+        // 택배 접수하기
 
+        if (checkSender && checkRecipient && checkProduct) {
 
-            // 입력된 데이터 기반으로 하나의 객체 생성
-            Parcel newParcel = new Parcel(Number, newsender, newrecipient, newproductinfo, "접수완료", 1000);
             // 생성된 개체를 배열에 저장
             repository.addParcelInformation(newParcel);
 
             // 택배 배열을 생성한 메모장 파일에 집어넣기
             repository.saveParcelArrayFile(newParcel);
 
-            return true;
-        } else if (selectExit.toUpperCase().contains("F")) {
-            return false;
-        } else {
-            return false;
+            System.out.println("택배가 접수되었습니다.");
         }
+    }
 
+
+    static void startInputParcel(){
+
+        inputSenderInfo();
+        inputRecipientInfo();
+        String makeT = makeTrackingNumber(inputSenderInfo(), inputRecipientInfo());
+        inputProductInfo();
+        Boolean checkS = checkSenderInfo(inputSenderInfo());
+        boolean checkR = checkRecipientInfo(inputRecipientInfo());
+        Boolean checkP = checkProductInfo(inputProductInfo());
+        Parcel saveP = saveParcel(makeT, inputSenderInfo(), inputRecipientInfo(), inputProductInfo());
+        receiptParcel(checkS, checkR, checkP,saveP);
```

Comment:
```suggestion
        Sender sender = inputSenderInfo();
        Recipient recipent = inputRecipientInfo();
        String makeT = makeTrackingNumber(sender, recipent);
        ProductInfo product = inputProductInfo();
        Boolean checkS = checkSenderInfo(sender);
        boolean checkR = checkRecipientInfo(recipent);
        Boolean checkP = checkProductInfo(product);
        Parcel saveP = saveParcel(makeT, sender, recipent, product);
        receiptParcel(checkS, checkR, checkP,saveP);
```
함수 호출한 각각의 반환값을 새로운 변수들에 할당한 후, 이용해야 할 것 같아요